src/sections/BlogTeaser/index.tsx

```diff
@@ -21,7 +22,7 @@ export type Post = {
   showExcerpt?: boolean;
 };
 
-const BlogTeaser: React.FC<{ posts: Post[] }> = (props) => {
+const BlogTeaser: React.FC<{ posts: any }> = (props) => {
```

Comment:
🙄

Reply:
i just realize we don't even use this component. 🤨 removed it for now